app/controllers/api/account.php

```diff
@@ -1498,6 +1498,12 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
                         'providerType' => MESSAGE_TYPE_EMAIL,
                         'identifier' => $email,
                     ]));
+
+                    $queueForEvents
+                        ->setEvent('users.[userId].create')
+                        ->setParam('userId', $user->getId())
+                        ->trigger();
```

Comment:
I think we also need to set payload here.

Endpoints to it by default - HTTP response body becomes the payload of the event.

You can then validate this by making function for this event, and printing `context.req.bodyText`. It should include entire user document (as JSON)

Reply:
Done